docs/reference/index.html

```diff
@@ -11,17 +11,6 @@ <h1 class = "content-header">Function Reference test</h1>
 
     <div class="doc-page-index">
     <ul data-gumshoe>
-      <li><a href="#section-spark-operations">Spark Operations</a></li>
```

Comment:
@javierluraschi In your previous pull request to rebuild HTML documents, what was the command or the option to select in RStudio that would tell `pkgdown` to render all the section elements here?

For some reason my attempt to update sparklyr HTML reference would always cause all of those elements to disappear, and I'm trying to figure out why :sweat_smile: 

I would really like to document this in the sparklyr Github wiki page as well for future reference.

Reply:
I think we just need to run `pkgdown::build_site()` from `sparklyr` and then something else from `spark.rstudio.com`. Honestly, we should also consider recreating the website entirely, we keep the content, but rebuild it in whichever way you want. The LF AI at some point also suggested having the docs also under docs.sparklyr.ai so maybe something we can consider if the current process gets hard to reverse engineer.